config/v2_2/translate_test.go

```diff
@@ -161,15 +161,15 @@ func TestTranslateFromV2_1(t *testing.T) {
 									Size:     100,
 									Start:    50,
 									TypeGUID: "HI",
-									GUID:     "4F68BCE3-E8CD-4DB1-96E7-FBCAF984B709",
+									GUID:     "$uuid36",
```

Comment:
unit tests (i.e. tests not under the `/tests` directory are not run with the BB test runner, so they wont be replaced unless you modify the end of this function to do it there). I think that would be great, but it's seperable from the bb tests and should be either another commit or another PR.